### PR TITLE
[Coop] GetStackTraces.

### PR DESCRIPTION
**Pull request by jaykrell**

### CODE REVIEW

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mono/metadata/threads.c [hunk 1]
```diff
@@ -4094,20 +4094,25 @@ mono_threads_perform_thread_dump (void)
...
 	guint32 thread_array [128];
 	MonoDomain *domain = mono_domain_get ();
 	MonoDebugSourceLocation *location;
 	int tindex, nthreads;
-
-	error_init (error);
 	
-	*out_threads = NULL;
-	*out_stack_frames = NULL;
+	MonoArray* out_threads = NULL;
```
Comment (jaykrell) [inline mono/metadata/threads.c]: Maybe remove `out_` from names?

**Comment by jaykrell** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by jaykrell** [comment]
@monojenkins build Linux i386

**Comment by lewing** [comment]
Looks like the lanes are stuck

**Comment by jaykrell** [comment]
@monojenkins build

**Comment by jaykrell** [comment]
@monojenkins build failed

**Pull request merged**